### PR TITLE
Javascript2 - Week3 Homework

### PR DESCRIPTION
**Pull request by Yasser2H**

### CODE REVIEW

File: javascript/javascript2/week3/fastest-presser/script.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            startButton.innerHTML = "Finished";
+            finishGame();
+            console.log("Timeout!");
+        } else {
+            startButton.innerHTML = timeRemaining + " Seconds";
+            // startButton.style.background = "#908f8f";
+            // startButton.style.color = "#535353";
+            // startButton.style.borderColor = "#000000";
+            // startButton.style.boxShadow = "#dcdcdc";
+            // startButton.style.textShadow = "#dcdcdc";
```
Comment (DeWinther) [inline javascript/javascript2/week3/fastest-presser/script.js:21]: Comments like this should just be removed

File: javascript/javascript2/week3/fastest-presser/script.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+let startButton = document.getElementById("btn-start-game");
+let fullPage = document.getElementsByTagName('body')[0];
+
+let leftPlayerCurrentScore = 0;
+let rightPlayerCurrentScore = 0;
+
+function startTimer(timeInSeconds) {
+    let timeRemaining = timeInSeconds;
+    let gameTimer = setInterval(function () {
```
Comment (DeWinther) [inline javascript/javascript2/week3/fastest-presser/script.js:9]: Should be a const

File: javascript/javascript2/week3/fastest-presser/script.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            // startButton.style.borderColor = "#000000";
+            // startButton.style.boxShadow = "#dcdcdc";
+            // startButton.style.textShadow = "#dcdcdc";
+        }
+        timeRemaining--;
+    }, 1000);
+}
+
+function startGame() {
+    let gameLength = document.getElementById("input-seconds").value;
```
Comment (DeWinther) [inline javascript/javascript2/week3/fastest-presser/script.js:28]: Should be a const

File: javascript/javascript2/week3/fastest-presser/script.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        countPresses(event.key.toLowerCase());
+    });
+
+    if (gameLength !== "") {
+        // document.getElementById("btn-start-game").disabled = true;
+        document.getElementById("input-seconds").value = "";
+        startTimer(gameLength);
+    } else {
+        alert("Please enter a time in seconds between 5 and 60")
+    }
```
Comment (DeWinther) [inline javascript/javascript2/week3/fastest-presser/script.js:40]: Your alert and if statement doesn't match, right now it's possible to input a number less than 5 and the game will still start

File: javascript/javascript2/week3/fastest-presser/script.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+
+function finishGame() {
+    fullPage.removeEventListener("keydown", function () {
+        console.log("Time is up!")
+    });
```
Comment (DeWinther) [inline javascript/javascript2/week3/fastest-presser/script.js:47]: Your event listener is never removed correctly, the `keydown` event is still being registered after the game has ended

File: javascript/javascript2/week3/fastest-presser/script.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (DeWinther) [inline javascript/javascript2/week3/fastest-presser/script.js:80]: Unneeded empty lines

File: javascript/javascript2/week3/script.js [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+planetLogFunction(logOutEarth);
+planetLogFunction(logOutSaturn);
+
+// Create a button with the text called "Log location". When this button is clicked the location (latitude, longitude) of the user should be logged out using this browser api
+const btnLogLocationTask = document.querySelector("#btn-log-location-task");
+btnLogLocationTask.addEventListener("click", getCurrentLocation);
+
+// Optional Now show that location on a map using fx the Google maps API
+mapboxgl.accessToken = 'pk.eyJ1IjoieWFzc2VyOTQiLCJhIjoiY2txOGFteWN5MDNjMjJ1bHdzNm1ocmNoaiJ9.NhGK1OH7MU9Io-6k9EnM6g';
```
Comment (DeWinther) [inline javascript/javascript2/week3/script.js:48]: It's never a good idea to have tokens directly in the code

**Review by martinvintherp — COMMENTED** [review]
Really good job!

I didn't really have much in regards to the code, the only thing I really noticed is the game is not stopping after the timer has ended and there is no visual indication to show which player won the game.

The last thing I noticed was that you committed quite a few `.idea` files, those should not be committed, because those a specific to your system. Take a look at the [.gitignore](https://git-scm.com/docs/gitignore) file so see how you can exclude files or even folders from git.

File: javascript/javascript2/week3/fastest-presser/script.js (hunk 6 above)
Comment (Yasser2H) [reply javascript/javascript2/week3/fastest-presser/script.js:80]: I always like to add extra lines at the bottom so the code moves to the middle of the screen :D

**Pull request merged**